src/commands/channels/activate.ts

```diff
@@ -0,0 +1,64 @@
+/*
+  * Copyright 2022, Nuance, Inc. and its contributors.
+  * All rights reserved.
+  *
+  * This source code is licensed under the Apache-2.0 license found in
+  * the LICENSE file in the root directory of this source tree.
+  */
+
+import {flags} from '@oclif/command'
+import chalk from 'chalk'
+import makeDebug from 'debug'
+
+import * as ChannelsAPI from '../../mix/api/channels'
+import * as MixFlags from '../../utils/flags'
+import {ChannelsActivateParams} from '../../mix/api/channels-types'
+import {MixClient, MixResponse} from '../../mix/types'
+import MixCommand from '../../utils/base/mix-command'
+import {DomainOption} from '../../utils/validations'
+
+const debug = makeDebug('mix:commands:channels:activate')
+
+export default class ChannelsActivate extends MixCommand {
+  static description = `activate a channel
+  
+  Use this command to activate a channel in a project.`
+
+  static examples = [
+    'mix channels:activate -P 1922 \\',
+    '  --channel bc40667c-e0f6-11ec-9d64-0242ac120003',
+  ]
+
+  static flags = {
+    project: MixFlags.projectWithDefaultFlag,
+    channel: flags.string({
```

Comment:
add "C" as short form to be consistent with other channels commands. Perhaps it's time to for channel have its own entry in flags.ts?

Reply:
Actually, following our conversation regarding "nice inputs", I suggest we be forward looking. What about we rename this flag to `--channel-id` with no short form.  We then can keep `--channel` with the short form `-C` for when we support using the channel name as unique identifier.  `--channel-id` would still be supported for backward compatibility (and to skip the extra endpoint call).

Reply:
Good point. Once everything is merged into the topic branch, let's take another look and see if we can refactor the flags.